cmake/FindHostProtoc.cmake

```diff
@@ -13,20 +13,32 @@ include_guard(GLOBAL)
 find_program(HOST_PROTOC_COMMAND "protoc" NO_CMAKE_FIND_ROOT_PATH)
 mark_as_advanced(HOST_PROTOC_COMMAND)
 
-if(HOST_PROTOC_COMMAND)
-  execute_process(
-    COMMAND ${HOST_PROTOC_COMMAND} --version
-    OUTPUT_VARIABLE protoc_output
-  )
-  string(STRIP "${protoc_output}" protoc_output)
-  string(REGEX REPLACE
-    "^libprotoc +([0-9.]+)$" "\\1" PROTOC_VERSION ${protoc_output})
-  message(STATUS "Found protoc: ${HOST_PROTOC_COMMAND} (version found \"${PROTOC_VERSION}\")")
-  unset(protoc_output)
-else()
+if(NOT HOST_PROTOC_COMMAND)
```

Comment:
To clarify, this is only a fatal error when building against protobufs target? Any other cmake error for regular builds should not result in a fatal error.

Reply:
It's a fatal error for P4 Control Plane as well. We have to use the host Protobuf compiler to compile our .proto files, regardless of whether we're compiling for the host platform or the ACC.

We should be okay. It's been a fatal error for a long time now. I just changed the logic from
```cmake
if (HOST_PROTOC_COMMAND)
  good stuff
else()
  message(FATAL_ERROR)
endif()
```

to

```cmake
if(NOT HOST_PROTOC_COMMAND)
  message(FATAL_ERROR)
endif()

good stuff
```

Reply:
ok. Since networking-recipe also heavily depends on the protobuf compilation, I guess that makes sense.